src/components/pages/playlists/PlaylistPlayer.vue

```diff
@@ -209,12 +209,13 @@
           :full-screen="fullScreen"
           :is-environment-skybox="isEnvironmentSkybox"
           :is-wireframe="isWireframe"
-          :preview-url="currentPreviewDlPath"
+          :preview-url="isCurrentPreviewModel ? currentPreviewDlPath : null"
           :style="{
             position: isComparisonOverlay ? 'absolute' : 'static',
             opacity: overlayOpacity
           }"
-          v-if="isCurrentPreviewModel && !isLoading"
+          @model-loaded="onModelLoaded"
+          v-show="isCurrentPreviewModel && !isLoading"
```

Comment:
why not keep `v-if` for performance purposes?

Reply:
Because I can't keep the reference to the object-viewer while using v-if.